docs/syntax/design-guidelines.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Design Guidelines
+section: Syntax
+permalink: /docs/syntax/design-guidelines/
+site_section: about-origami
+---
+
+# Design guidelines workflow
+
+Design guidelines can be added to any component in the registry and can be written in Markdown format. Guidelines will appear below the component title and above the component demos.
+This guide will take you through how to add design guidelines to a component using Github, by the end of the tutorial, you should be able to create, write, and submit a pull request to add the guidelines to the component.
```

Comment:
Super super pedantic but the "H" in GitHub is capitalised, few of these dotted around